pkg/ddl/executor.go

```diff
@@ -6154,18 +6154,22 @@ func (e *executor) CreateCheckConstraint(ctx sessionctx.Context, ti ast.Ident, c
 		return err
 	}
 	job := &model.Job{
+		Version:        model.GetJobVerInUse(),
 		SchemaID:       schema.ID,
 		TableID:        tblInfo.ID,
 		SchemaName:     schema.Name.L,
 		TableName:      tblInfo.Name.L,
 		Type:           model.ActionAddCheckConstraint,
 		BinlogInfo:     &model.HistoryInfo{},
 		CDCWriteSource: ctx.GetSessionVars().CDCWriteSource,
-		Args:           []any{constraintInfo},
 		Priority:       ctx.GetSessionVars().DDLReorgPriority,
 		SQLMode:        ctx.GetSessionVars().SQLMode,
 	}
 
+	job.FillArgs(&model.AddCheckConstraintArgs{
```

Comment:
you can pass args in JobWrapper now, see https://github.com/pingcap/tidb/blob/b1767ca12464c45f741ace20b9ec679d2ea4de5f/pkg/ddl/job_scheduler.go#L732